modules/plugins/languages/rust.nix

```diff
@@ -145,13 +145,13 @@ in {
               on_attach = function(client, bufnr)
                 default_on_attach(client, bufnr)
                 local opts = { noremap=true, silent=true, buffer = bufnr }
-                vim.keymap.set("n", "<leader>rr", ":RustLsp runnables<CR>", opts)
-                vim.keymap.set("n", "<leader>rp", ":RustLsp parentModule<CR>", opts)
-                vim.keymap.set("n", "<leader>rm", ":RustLsp expandMacro<CR>", opts)
-                vim.keymap.set("n", "<leader>rc", ":RustLsp openCargo", opts)
-                vim.keymap.set("n", "<leader>rg", ":RustLsp crateGraph x11", opts)
+                vim.keymap.set("n", "<localleader>rr", ":RustLsp runnables<CR>", opts)
```

Comment:
add an option to set <localleader>

also pls document breaking change

Reply:
We should actually remove vim.leaderKey as well, and encourage users to use vim.globals instead.